src/pages/patterns/loaders.mdx

```diff
@@ -0,0 +1,249 @@
+is used to show a blank stand-in version of the component that has not fully loaded. It provides a visual estimate of
+the space needed. This is the preferred loader in most situations as it provides a sense of progress, helps orient the
+user, and sets expectations on the type of content they can expect.
+
+![Example of skeleton loaders for predictable content](figma:patterns-loaders-skeleton.png)
+
+The size of the loader should approximately reflect the content it represents. Show a few skeleton loaders to orient
+the user, but don’t map out every element even if a page has a lot of content. Mapping out skeleton loaders in the
+first few elements of the header/body area are enough to convey the loading process.
+
+Loader animation has a built-in short delay which should give time for the fast loading content to show before the
+loader becomes visible. Loader should be persistent until the content is fully loaded.
+
+Use default skeleton loaders on white or light backgrounds. Use light skeleton loaders on dark backgrounds.
+
+<BestPractice>
+  <Do
+    imageIsFreeForm
+    imageSource={props.data.LoadersSkeletonDo.childFile.childImageSharp.gatsbyImageData}
+    alt="A few skeleton loaders to orient the user"
+  >
+    <ul>
+      <li>Show a skeleton loader to orient the user.</li>
+    </ul>
+  </Do>
+  <Dont
+    imageIsFreeForm
+    imageSource={props.data.LoadersSkeletonDont.childFile.childImageSharp.gatsbyImageData}
+    alt="Too many skeleton loaders"
+  >
+    <ul>
+      <li>Avoid mapping every line with a skeleton loader.</li>
+    </ul>
+  </Dont>
+</BestPractice>
+
+### Unpredictable content
+
+A <a href="/components/spinner">Spinner loader</a> is used when content is unknown or unpredictable. Spinner appears
+before a page is loaded to assure a user that the content is on its way. Use spinners for pages and screens or within
+components such as tables. Center the spinner in the container or content you’re applying it to. Spinner is also used
+as an alternative loader to Skeleton in situations where the content can't be easily estimated but it's still
+important to provide a sense of progress.
+
+![Example of Spinner loader for unpredictable content](figma:patterns-loaders-spinner.png)
+
+### Completing action
+
+The <a href="/components/dots">Dots loader</a> communicates ongoing activity after a user takes an action. It tells
+them that something is taking place. Dots loader is used for buttons, search inputs, or other small components to
+indicate these components are doing something.
+
+Use the loader for actions where it might take several seconds for the process to complete.
+
+![Example of dot loaders when completing an action](figma:patterns-loaders-completing-action.png)
+
+1. <a href="/components/button#type">Primary buttons</a> are common candidates for using this loader
+1. <a href="/components/icon-button">Icon buttons</a> are less likely to use this loader but still necessary
```

Comment:
Is this supposed to be 2?
```suggestion
2. <a href="/components/icon-button">Icon buttons</a> are less likely to use this loader but still necessary
```

Reply:
Fun fact - they can all be 1 for easier maintenance. https://www.markdownguide.org/basic-syntax/#ordered-lists

> The numbers don’t have to be in numerical order, but the list should start with the number one.

Reply:
Good to know! Thanks, @smritimadan.